Makefile

```diff
@@ -234,10 +245,42 @@ l1:
 	go get github.com/jstemmer/go-junit-report; \
 	mkdir -p out; \
 	go test -v -coverpkg=github.com/theotw/natssync/pkg/... -coverprofile=out/unit_coverage.out github.com/theotw/natssync/pkg/... > out/l1_out.txt 2>&1 || SUCCESS=1; \
-	cat out/l1_out.txt | go-junit-report > out/report_l1.xml || echo "Failure generating report xml"; \
+	cat out/l1_out.txt | go-junit-report > out/l1_report.xml || echo "Failure generating report xml"; \
 	cat out/l1_out.txt; \
 	exit $$SUCCESS;
 
+deploysingle: export SYNCCLIENT_PORT=8083
+deploysingle: export TEST_MODE=false
+deploysingle:
+	./single_cluster_test/docker-cleanup.sh
+	TEST_MODE=${TEST_MODE} SYNCCLIENT_PORT=${SYNCCLIENT_PORT} ./single_cluster_test/docker-deploy.sh "${IMAGE_TAG}"
+
+integrationtests: export SYNCCLIENT_PORT=8083
+integrationtests:
+	go run apps/natstool.go -u nats://localhost:4222 -s natssync.registration.request -m '{"authToken":"42","locationID":"client1"}'
+	sleep 5
+	curl -X POST -H 'Content-Type: application/json' -d '{"authToken":"42","locationID":"client1"}' "http://localhost:${SYNCCLIENT_PORT}/bridge-client/1/register" | jq .locationID | sed s/\"//g > locationID.txt
+	echo "ID: `cat locationID.txt`"
+	go run apps/echo_client.go -m "hello world" -i `cat locationID.txt`
+	syncserver_url='http://localhost:8080' syncclient_url='http://localhost:${SYNCCLIENT_PORT}' natsserver_url='nats://localhost:4222' go test -v github.com/theotw/natssync/tests/integration/...
+	curl -i -f -X POST -H 'Content-Type: application/json' -d '{"authToken":"42","locationID":"`cat locationID.txt`"}' "http://localhost:${SYNCCLIENT_PORT}/bridge-client/1/unregister"
+	echo "Unregistered ID: `cat locationID.txt`"
+	echo "Single cluster test done"
```

Comment:
This was carried over from `cicd.yaml` practically unchanged. I thought about putting it in its own script but I think it's small enough that there's no need yet.